### PR TITLE
Feature 1 complete - Bryan Valdovinos

### PR DESCRIPTION
**Pull request by bryanvaldovinos**
Screen Shot 2022-05-31 at 3 10 51 PM
Screen Shot 2022-05-31 at 3 13 30 PM

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  padding: 7px;
+  border-radius: 7px;
+  border-width: 1px;
+}
+
+.input-style-2 {
+  padding: 7px;
+  border-radius: 7px;
+  border-width: 1px;
+}
```
Comment (codingcodymiller) [inline css/styles.css]: This is exactly the same as the rule above it, it should be deleted 👀

File: index.html [hunk 2]
```diff
@@ -11,15 +11,55 @@
...
 </head>
-<body>
-  <header>
-  </header>
-  <main>
-  </main>
+<body class="body-bgc">
+  <div class="row header-bgc">
+    <header class="column-full">
+      <div class="row container">
```
Comment (codingcodymiller) [inline index.html]: You shouldn't have `container`s inside of rows or columns. You should make the `container` the most external layer here, right inside the element you apply the purple background color to. That way your purple background fills the whole width of the viewport, but the content is confined in the `container`

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  data.nextEntryId++;
+  contact.reset();
+  img.setAttribute('src', 'images/placeholder-image-square.jpg');
+
+}
+
+function local(event) {
+  var entriesJSON = JSON.stringify(data);
+  localStorage.setItem('Data Info:', entriesJSON);
+}
```
Comment (codingcodymiller) [inline js/main.js]: Please keep all localStorage-related code in the `data.js` file 🙏

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by bryanvaldovinos** [comment]
Screen Shot 2022-05-31 at 3 52 53 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**